sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -4,10 +4,11 @@
 
```

Comment:
Some of the contracts show output_2, once all API are addressed, it should be addressed.

Reply:
Yeah that makes sense. I think it has to do with how APIExtractor handles name conflicts which may mean we're importing or exporting the same thing twice. I can squint for a bit and assume it'll get resolved once everything else is updated 👍 

But if you run into trouble or get stuck on it I can take a closer look